en/option/series/gauge.md

```diff
@@ -295,6 +295,12 @@ Gauge chart pointer.
 
 Whether to show the pointer.
 
+### showAbove(boolean) = true
```

Comment:
You have misunderstood. It's not changelog you need to add. You should add `{{ use: partial-version( version = "5.2.0" ) }}` at here, there are serval examples in `gauge.md`